apps/custom-config/wakeup-effect.js

```diff
@@ -254,12 +256,16 @@ class WakeupEffect extends BaseConfig {
           if (typeof queryObj.wakeupSoundEffects !== 'object') {
             return
           }
-          clearDir(ActivationConfig.customPath).then(() => {
-            downloadWav(queryObj.wakeupSoundEffects, ActivationConfig.customPath).then((fileList) => {
-              this.notifyActivation(fileList)
-            }).catch((err) => {
-              logger.warn(`download custom wakeup sound error: ${err}`)
-            })
+          logger.error(ActivationConfig.customPath)
+          mkdirpAsync(ActivationConfig.customPath).then(() => {
+            return clearDir(ActivationConfig.customPath)
+          }).then(() => {
+            logger.error('download')
```

Comment:
Why do you use error to print "download"?

Reply:
updated.